ackermann_steering_controller/package.xml

```diff
@@ -32,8 +32,9 @@
   <test_depend>controller_manager</test_depend>
   <test_depend>gazebo_ros</test_depend>
   <test_depend>geometry_msgs</test_depend>
-  <test_depend>rosunit</test_depend>
   <test_depend>rostest</test_depend>
+  <test_depend>rosunit</test_depend>
+  <test_depend>rviz</test_depend>
```

Comment:
I think it is better to not depend on rviz. It should not be crucial for the test.

Reply:
And they should already be behind a `show_rviz` or `gui` flag.

Reply:
removed rviz at https://github.com/ros-controls/ros_controllers/pull/380/commits/e745c484517543b00b25111f5aa9dfe749c236ed